src/apphosting/index.ts

```diff
@@ -122,7 +122,7 @@ export async function doSetup(
 
   if (!confirmRollout) {
     logSuccess(`Successfully created backend:\n\t${backend.name}`);
-    logSuccess(`Your site will be deployed at:\n\thttps://${backend.uri}`);
+    logSuccess(`Your backend will be deployed at:\n\thttps://${backend.uri}`);
```

Comment:
@tonyjhuang the wording "site" feels odd because we aren't using that anywhere else. I think "app" or "backend" would make more sense. Let me know what you think.

Reply:
backend sgtm